### PR TITLE
Update touchpad tuning guide

### PR DESCRIPTION
**Pull request by zhenyasav**

### CODE REVIEW

**Comment by themar-msft** [comment]
Hi @zhenyasav - Thanks for the contributions! Can you please sign the Contributor License Agreement so we can publish these changes?

Thanks!

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 1]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+> Be aware that serious problems can occur if the registry is modified incorrectly. Ensure these steps are followed carefully. For additional protection, back up the registry before modification. See [how to back up and restore the registry in Windows](https://support.microsoft.com/en-us/help/322756/how-to-back-up-and-restore-the-registry-in-windowshttps:/support.microsoft.com/en-us/help/322756/how-to-back-up-and-restore-the-registry-in-windows).
 
-## Windows 10 tuning capabilities
+## Servicing your configuration
+As of 19H1 Windows supports two locations in the registry for customizing the touchpad. The keys in the rest of this document are located in one of these two places. This makes it possible to service your configuration through Windows Update.
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: Remember to replace 19H1 with whatever the correct name is

Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: May want to check what other pages use for registry nomenclature. Technically, HKLM\…\PrecisionTouchpad is a key, e.g. RightClickZoneWidth is a value, and 50 is data.

Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: You should elaborate on "This" for servicing - perhaps only mention it after the HKR key is described. The fact that we support two locations isn't what enables WU, it's what the new location is :)

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 2]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+## Servicing your configuration
+As of 19H1 Windows supports two locations in the registry for customizing the touchpad. The keys in the rest of this document are located in one of these two places. This makes it possible to service your configuration through Windows Update. 
+- The device's `HKR` hardware key:
+    - [Introduction to registry keys for drivers](https://docs.microsoft.com/en-us/windows-hardware/drivers/wdf/introduction-to-registry-keys-for-drivers)
+    - [Device Driver INF AddReg Directive](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/inf-addreg-directive)
+    - [Opening a Device's Hardware Key](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/opening-a-device-s-hardware-key)
+- The old location: `HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\PrecisionTouchPad`
 
-Windows 10, version 1607 provides new registry capabilities for better device tuning. Note that the default values are set to work well with most devices. 
+For _each_ of the registry keys in the rest of this guide, the HKR location is checked first, then the HKLM location next, enabling the ability to split configuration between the two.
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: perhaps say that this allows the HKR location to serve as an override on a per-value basis?

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 3]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+## Accidental Activation Prevention
+AAP is a feature of Windows that suppresses certain actions like mouse moves and taps for a short time after keyboarding events. The specific timeouts depend on the type of action and the region of the touchpad it was performed, as well as the user setting called `Touchpad sensitivity` found in Touchpad settings. More sensitive means less time is spent suppressing input after keyboarding events. Maximum sensitivity means AAP timeouts are zero and AAP is not in effect.
+
+| | |
+| --- | --- |
+| ![A screenshot of Touchpad settings in Windows](../images/aap-sensitivity.png) | ![A screenshot of Touchpad sensitivity options in Windows settings](../images/aap-sensitivity-open.png) |
+
+## Tunable parameters
+Windows defines two classes of "Curtain" regions on the touchpad to help prevent accidental input when users are resting their hands on the keyboard or touchpad during normal use. This splits the touchpad into three regions:
+- High sensitivity region: no input is ever suppressed in this region
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: Not technically true - it seems like taps, for instance, are suppressed when the AAP timer is active.

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 4]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+- Curtain region: input may be suppressed for some time in this region
+- Supercurtain region: certain kinds of input like taps are almost always suppressed when the gesture interacts with this region
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: Supercurtain is actually even stronger - input initiated in this area is IGNORED until it leaves the area, except under some very specific conditions (e.g., if you perform a 2-finger pan with one finger in the super curtains, we may allow it).

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 5]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+
+As of 19H1 there are two ways to configure the Curtain region. Directly from the edges of the touchpad, or the legacy method where curtains are determined based on the location of the touchpad with respect to the keyboard.
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: Grammar

Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: The legacy way isn't explicitly configuring the curtain - I'd say something like:

In 19H1, the curtain region's dimensions can be explicitly configured. In previous releases, it was configured implicitly based on the location of the touchpad with respect to the keyboard.

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 6]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+
+Regardless of which curtain method is used, it is strongly recommended to describe to Windows the physical touchpad offset relative to the keyboard via `SpaceBarOffset` and `HorizontalOffset`. This is necessary for Windows to make decisions around a consistent and predictable user experience when the user switches between keyboarding and pointing. For more information, see the [Windows Precision Touchpad Device Integration](https://docs.microsoft.com/en-us/windows-hardware/design/component-guidelines/windows-precision-touchpad-device-integration) guide.
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: I'd rephrase in combination with above - e.g., even when explicitly configuring the curtains, it is still strongly recommended to provide the positioning information.

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 7]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
+| Registry Key | Type | Units | Range | Default | Availability |
+|--- | --- | --- | --- | --- | --- |
+| SuperCurtainTop | DWORD | Himetric | 0+ | 100 | TH2+ |
+| SuperCurtainLeft | DWORD | Himetric | 0+ | 100 | TH2+ |
+| SuperCurtainRight | DWORD | Himetric | 0+ | 100 | TH2+ |
+| SuperCurtainBottom | DWORD | Himetric | 0+ | 0 | 19H1+ |
 
-The restricted sensitivity zone is the lowest sensitivity region of the touchpad, and is designed for the region of the touchpad where users would not leave fingers during intentional use. Note these are very small by default and should only be widened for very large devices. 
+### Right-click Zone
+The following registry keys allow the configuration of the button row and right click zone. The button row containing the right and left buttons will be controlled by `RightClickZoneHeight` from the bottom edge of the touchpad. For a right-handed configuration, the meaning of 0,0 for these settings is the bottom right corner of the touchpad. For a left handed configuration, the reference is in the bottom left corner.
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: 0 isn't an allowed value. And what do you mean by "button row"?

File: design/component-guidelines/touchpad-tuning-guidelines.md [hunk 8]
```diff
@@ -9,54 +9,105 @@ ms.date: 05/02/2017
...
 
-The top, left, and right zones' sizes are defined independently as a distance from their respective edge. They use the registry keys shown in the following table. 
+| Registry Key | Type | Units | Range | Default | Availability |
+|--- | --- | --- | --- | --- | --- |
+| RightClickZoneHeight | DWORD | Percent | 1-100 | 25 | TH1+ |
+| RightClickZoneWidth | DWORD | Percent | 1-100 | 50 | TH1+ |
+
+### Suppressing Accidental Activation Protection
+
+Setting the following key is equivalent to the user choosing "Most sensitive", which disables AAP. When this key is set, the `Touchpad sensitivity` chooser may be set to anything in Settings, but has no effect.
```
Comment (rossnichols) [inline design/component-guidelines/touchpad-tuning-guidelines.md]: Chooser?
